### DISCUSSION TITLE
Workflow on PR labeled action always fails if there is already OK workflow for the same commit

### DISCUSSION THREAD

**Discussion by aindlq** [community/community#25568] [Actions]
<p>Workflow on pull_request labeled action always fails if there is already completed workflow for the same commit.</p>
<p>Failed workflow is displayed in “Actions” tab but is not even accessible, click on the failed workflow (that was triggered by labeled action) always opens completed workflow (that was triggered by open or synchronize action).</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/b/b6a0fc43f09e54f969c7dca411e3a565a33245c0.png" data-download-href="/uploads/short-url/q3BOaR1ph0SSkPmxVZG75LsGely.png?dl=1" title="Screenshot_2019-11-20 researchspace researchspace.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/2255044/181091843-90389a43-535f-4160-8247-caa1ad68dffa.png" alt="Screenshot_2019-11-20 researchspace researchspace.png" data-base62-sha1="q3BOaR1ph0SSkPmxVZG75LsGely" width="690" height="95" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/b/b6a0fc43f09e54f969c7dca411e3a565a33245c0_2_690x95.png, https://aws1.discourse-cdn.com/github/optimized/2X/b/b6a0fc43f09e54f969c7dca411e3a565a33245c0_2_1035x142.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/b/b6a0fc43f09e54f969c7dca411e3a565a33245c0_2_1380x190.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/b6a0fc43f09e54f969c7dca411e3a565a33245c0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_2019-11-20 researchspace researchspace.png</span><span class="informations">1856×257 27.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/a/a13321879fbba3c6d253c50cfddcfc45d4cd0fb8.png" data-download-href="/uploads/short-url/n02q3PM6JVbeL0wR79Mm1IeTzOE.png?dl=1" title="Screenshot_2019-11-20 researchspace researchspace(1).png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/2255044/181091844-712d98da-4907-40aa-8c99-0bb852b809f6.png" alt="Screenshot_2019-11-20 researchspace researchspace(1).png" data-base62-sha1="n02q3PM6JVbeL0wR79Mm1IeTzOE" width="690" height="464" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/a/a13321879fbba3c6d253c50cfddcfc45d4cd0fb8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_2019-11-20 researchspace researchspace(1).png</span><span class="informations">692×466 8.62 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>see config <a href="https://github.com/researchspace/researchspace/blob/github-actions-ci/.github/workflows/main.yml" rel="nofollow noopener">https://github.com/researchspace/researchspace/blob/github-actions-ci/.github/workflows/main.yml</a></p>

**Comment by BrightRan** [comment, marked as answer]
<p>H <a class="mention" href="/aindlq">@aindlq</a> ,</p>
<p>The workflow run triggered by PR labeled to be failed is not caused by the previous completed workflow run, the root cause is an incorrect syntax to get the label name on the if condition.<br>
The correct syntax should be:</p>
<pre><code>if: github.event.action != 'labeled' || github.event.label.name == 'build_image'
</code></pre>

**Comment by aindlq** [comment]
<p><a class="mention" href="/brightran">@brightran</a>  thanks, that fixed the problem?</p>
<p>What confused me is that it was not possible to access failed build at all, it always redirected to the successful one.</p>
